mkdocs.yml

```diff
@@ -0,0 +1,10 @@
+site_name: PyCatcher Documentation
```

Comment:
Mkdocs file is responsible to keep the documentation together and will deploy the config to the live site that we set up